integration_tests/models/schema.yml

```diff
@@ -234,13 +234,31 @@ sources:
     schema: "{{ target.schema }}"
     tables:
       - name: groups
+        columns:
+          - name: group_a
+            data_type: "{{ 'STRING' if target.type == 'bigquery' else 'character varying' if target.type == 'redshift' else 'TEXT' }}"
```

Comment:
I need the data type to be the exact type in the information schema for each platform :/

Reply:
Have you considered `data_type: "{{ elementary.type_string() }}"`?

Reply:
`type_string` won't be enough for me as it's not exactly the same as what I'll find in INFORMATION_SCHEMA.
But extracting to a macro absolutely makes sense, I'll do it.

Reply:
Hmm seems like macros are not supported in yamls and I think you can't add "{% %}" sections there either (I think the yaml is parsed before dbt expands specific settings with jinja)
If you have another idea that's less ugly than what I currently did will be happy to hear :)

Reply:
Apologies for misleading, I recalled this is supported.